mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -739,6 +746,7 @@ impl SaplingSyncLoopHandle {
 
             let wallet_ops_guard = wallet_db.get_update_ops().expect("get_update_ops always returns Ok");
             let scan = DataConnStmtCacheWrapper::new(wallet_ops_guard);
+            info!("START SCANNING WALLETDB");
```

Comment:
Please provide more context in this log message.

Reply:
ok